docker/php-7.4/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM php:7.4-cli-alpine3.12
```

Comment:
do we need to specify exact alpine version?

Reply:
It's best to avoid getting image build failures randomly when new versions are published. The down side is that this must be updated manually almost every time we want to use those new versions.

Reply:
```suggestion
FROM php:7.4-cli-alpine3.13
```